doc/base/classes.xml

```diff
@@ -22390,7 +22390,19 @@
 			<argument index="0" name="rel_vec" type="Vector3">
 			</argument>
 			<description>
-				Move the body in the given direction, stopping if there is an obstacle. The returned vector is how much movement was remaining before being stopped.
+				Move the body in the given direction, stopping if there is an obstacle. If as a result of a movement there will be any collision then informations about this collision will be in returned dictionary.
+				Dictionary will contains those keys:
+				"position" - collision position
```

Comment:
All those lines will be treated as separate paragraph, it will look quite bad in the help.
At least for the rst parser, you can prefix the lines with `- ` so that they're treated as an unordered list. For the in-editor help, not sure how it will turn out, maybe a [list] tag should be added.